src/bundler.js

```diff
@@ -136,7 +138,7 @@ export async function bundleProd({
 			return fileName;
 		},
 		hoistTransitiveImports: true,
-		plugins: [terser({ compress: true, sourcemap })],
+		plugins: [minify && terser({ compress: true, sourcemap })],
```

Comment:
A thought: we could allow `minify` to be an object in addition to a Boolean, with the format:

```js
{ terser: Partial<TerserConfig>, css: Partial<CSSNanoConfig> }
```

Reply:
Personally that feels a bit too deep, but happy to add it if desired. 

In my stuff, I add separate `cssnano` and/or `terser` config blocks so that they're separately exportable. But then again, I always use the `@pwa/cli` (and now `freshie`) format where every config key is top-level.